### DISCUSSION TITLE
Support background image URLs in JIT mode

### DISCUSSION THREAD

**Discussion by dannote** [tailwindlabs/tailwindcss#4880] [Ideas]
Since JIT now supports things like `text-[var(--mystery-var)]`, maybe it's time to add something like `bg-[url(/image.jpg)]`?

**Comment by paultman** [comment]
I am also hoping for a similar ability, to inject custom image url's via JIT rather than via the config file.

I noticed simply using an inline style attr for background-image in an example on the tailwinds page for something else, so I did that.  
However now I have another place where I need to change the url itself for responsive.  

Would be nice to have something like md:bg-image-[url: headerM.png] and lg:bg-image-[url: headerL.png]
